lib/active_merchant/billing/gateways/reach.rb

```diff
@@ -0,0 +1,176 @@
+require 'base64'
```

Comment:
I don't think you need this since it's already included in the `ActiveMerchant` module.

Reply:
yes removing it 👍